app/src/main/java/com/keylesspalace/tusky/ViewMediaActivity.kt

```diff
@@ -137,7 +143,18 @@ class ViewMediaActivity : BaseActivity(), ViewImageFragment.PhotoActionsListener
             true
         }
 
-        window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LOW_PROFILE
```

Comment:
ViewMediaActivity does not work for me:

 - There's a huge lag when opening a picture
 - System bars are hidden by default (which is out of sync with viewer UI)
 - If I tap anywhere then system UI shows up. Bottom nav bar from system will be on top of the description
 - If I tap again then viewer UI disappears but system UI still shows

Reply:
I see, that sucks. I'll try to find a better solution or revert to the deprecated flag.

Reply:
I think reverting is the best solution for now.